src/modules/journal/__tests__/journal.selector.spec.ts

```diff
@@ -349,4 +455,16 @@ describe('JournalSelector', () => {
     });
   });
 
+  describe('hasSlotsAfterSelectedDate', () => {
+    it('should return TRUE if slots DO EXIST after journal selected date', () => {
+      expect(hasSlotsAfterSelectedDate(baseJournalData)).toEqual(true);
+    });
+
+    it('should return FALSE if slots DO EXIST after journal selected date', () => {
+      const journalWithoutSlotsAfterSelectedDate = cloneDeep(baseJournalData);
```

Comment:
use 
```
= {
...baseJournalData,
};
```
instead of `cloneDeep`

Reply:
There was a reason we chose cloneDeep over spread in this instance. Spread will only create a shallow copy and changes to nested objects in copies created using this method would therefore also undesirably update the original baseJournalData object.